From 1e4b53bf988ac6c47516f1fc3c8f912648cf6f45 Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Mon, 14 Nov 2016 10:38:07 +0000
Subject: [PATCH] protobuf/examples: fix QA warning for GNU_HASH

WARNING: protobuf-3.0.0-r0 do_package_qa: QA Issue: No GNU_HASH in the
elf binary: .../add_person_cpp, .../list_people_cpp

Upstream-Status: Pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 examples/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/examples/Makefile b/examples/Makefile
index 383e77c..9497105 100644
--- a/examples/Makefile
+++ b/examples/Makefile
@@ -32,11 +32,11 @@ protoc_middleman_go: addressbook.proto
 
 add_person_cpp: add_person.cc protoc_middleman
 	pkg-config --cflags protobuf  # fails if protobuf is not installed
-	c++ add_person.cc addressbook.pb.cc -o add_person_cpp `pkg-config --cflags --libs protobuf`
+	c++ add_person.cc addressbook.pb.cc ${LDFLAGS} -o add_person_cpp `pkg-config --cflags --libs protobuf`
 
 list_people_cpp: list_people.cc protoc_middleman
 	pkg-config --cflags protobuf  # fails if protobuf is not installed
-	c++ list_people.cc addressbook.pb.cc -o list_people_cpp `pkg-config --cflags --libs protobuf`
+	c++ list_people.cc addressbook.pb.cc ${LDFLAGS} -o list_people_cpp `pkg-config --cflags --libs protobuf`
 
 add_person_go: add_person.go protoc_middleman_go
 	go build -o add_person_go add_person.go
-- 
2.9.3

